bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapAPIHandler.java

```diff
@@ -74,7 +74,7 @@ public void initialize() {
         config = getConfigAs(OpenWeatherMapAPIConfiguration.class);
 
         boolean configValid = true;
-        if (config.apikey == null || config.apikey.trim().isEmpty()) {
+        if (config.apikey == null || config.apikey.isBlank()) {
```

Comment:
The API key is a required configuration parameter so it cannot be null.

```suggestion
        if (config.apikey.isBlank()) {
```